repo/packages/M/marathon/0/marathon.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
Just a side comment that I thought about now. Technically this are not valid JSON until the mustache template engine has been applied. Should we give them a different file name? Is there are standard way of doing this for Mustache?